### DISCUSSION TITLE
NVIDIA 自駕車技術

### DISCUSSION THREAD

**Discussion by jason79461385** [1101-GS4719-C/Python-Community-GS4719-C#445] [General]
1.透過Lidar用以建構出模型
2.藉由感測器抽象層（SAL）為不同的感測器提供一個統一的使用接口，提供傳感器原始數據序列化
3.感測器外掛程式（跟安裝在汽車上的感測器或者鏡頭連接）
4.深度神經網路(DNN)
  Drive AV:軟體堆疊包含感知、製圖和規劃層，以及各種利用實際駕駛資料進行訓練的多元深度神經網路。這些感知輸出可以用於自動駕駛和製圖功能，透過此運算模組可以讓車輛遠離危險
  Drive IX:是一個開放軟體平台，為創新的人工智慧駕駛艙解決方案提供內部感測。負責偵測駕駛人的狀況，例如：精神狀況。在駕駛人精神狀況不佳時會發出提醒。	

![image](https://user-images.githubusercontent.com/75102829/157393070-df3216b3-43f4-4175-aaec-6acea5fc2f74.jpeg)

參考網站：https://www.nvidia.com/zh-tw/self-driving-cars/

**Comment by xkllkx** [comment]
謝謝樓主的分享，看來解放雙手的日子應該不遠了！

**Comment by ThomasMeng0515** [comment]
補充一下光達(Lidar)是一個光學遙測技術，原理是利用一束脈衝雷射照向目標物以取得與目標物的距離的與其他相關參數。

**Comment by HuanYuuuuuu** [comment]
但自駕車的辨識周遭系統依然常常在燈光昏暗或是窄小的巷子時失靈，會把垃圾桶辨識成人之類的...
也許在燈光昏暗時還有更多方法可以彌補此缺點

**Comment by dayu1004** [comment]
除了自駕技術之外，我記得目前也有汽車車頭燈在偵測到人的時候會將直射人體的燈光(燈泡?)暫時關閉的設計，目的是為了不要讓人眼直視燈光造成不適感。不知道是不是也是利用Lidar去進行偵測的XD?

**Comment by ghost** [comment]
拿到駕照過了三年了
然後都沒碰過車，這東西該來了吧><

**Comment by yuhao112** [comment]
自己是有搭過有自駕車的經驗，雖然其自動化層級還沒那麼高，但在使用的時候其實就有遇到過問題:自動化的操作並不是那麼人性化。之前有開自動遵循車道，當偏離車道時，車子會自行強拉回來，乘客的感覺較易感到不適，因此我在文章上面提及遠離危險也怕是會以較不人道的方式實行

**Comment by etin2468** [comment]
這樣的意思是NVIDIA也要加入自駕車的戰場了嗎?不知道現在特斯拉的軟體跟這個比誰比較厲害

**Comment by yucheng1028** [comment]
總覺得還是要先解決法律上的限制，畢竟已經有很多車廠都做出level 3的自駕，但還是因法規而被限制

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，讓我對自駕技術有更進一步的認識。

**Comment by sean1325418** [comment]
現在自駕車技術還沒到完美，還有很多進步空間，期待未來可以上車直接睡覺，醒來已經到達目的地。

**Comment by cordydeng** [comment]
如果自駕車的技術達到了完美 , 那我們是不是就不需要考駕照了呢  , 又或者說我們再也不需要司機了呢?不知道大家有什麼看法

**Comment by Yuhsin000** [comment]
常常聽到NVIDIA，原來他們有這個技術，有在發展這個!

**Comment by PoooYen** [comment]
Level3要來了嘛!!!!!
